src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests.csproj

```diff
@@ -19,6 +19,9 @@
     <None Update="ExpectedConfigurations\CollectionRuleDefaults.json">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Update="ExpectedConfigurations\DotnetMonitorDebug.json">
```

Comment:
I'm not sure about this myself, but have you considered updating the test runners to enable the internal exceptions by default?